signature-aggregator/aggregator/aggregator.go

```diff
@@ -176,19 +174,14 @@ func (s *SignatureAggregator) CreateSignedMessage(
 		))
 	}
 
-	// TODO: remove this special handling and replace with ACP-118 interface once available
-	var reqBytes []byte
-	if sourceSubnet == constants.PrimaryNetworkID {
-		req := coreEthMsg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = coreEthMsg.RequestToBytes(coreEthCodec, req)
-	} else {
-		req := msg.MessageSignatureRequest{
-			MessageID: unsignedMessage.ID(),
-		}
-		reqBytes, err = msg.RequestToBytes(codec, req)
-	}
+	reqBytes := networkP2P.ProtocolPrefix(networkP2P.SignatureRequestHandlerID)
```

Comment:
Consider using `[p2p.PrefixMessage](https://github.com/ava-labs/avalanchego/blob/ab83fb41528de93c1790301cdd67a07dda9299f0/network/p2p/client.go#L79)` to construct `reqBytes`.

Reply:
Thanks for pointing this out. Addressed in the other PR https://github.com/ava-labs/awm-relayer/pull/463